### PR TITLE
Task 8

### PR DESCRIPTION
**Pull request by VladMironchuk**

### CODE REVIEW

File: src/elements/modal/signInModal.tsx [hunk 1]
```diff
@@ -21,9 +20,9 @@ const SignInModal: React.FC = () => {
     dispatch(userActions.toggleLogging());
   };
 
-  const onSignIn = () => {
-    dispatch(modalActions.toggleSignIn());
-  };
+  // const onSignIn = () => {
```
Comment (borodayev) [inline src/elements/modal/signInModal.tsx]: need to cleanup

File: src/elements/privateRoute/privateRoute.tsx [hunk 2]
```diff
@@ -13,18 +13,19 @@ const PrivateRoute: React.FC<Props> = (props) => {
   return (
     <Route
       path={path}
-      render={({ location }) =>
-        isLogged ? (
+      render={({ location }) => {
+        console.log(location);
```
Comment (borodayev) [inline src/elements/privateRoute/privateRoute.tsx]: do not push console.logs to the codebase

File: src/hooks/useHttp.tsx [hunk 3]
```diff
@@ -0,0 +1,50 @@
+import { useCallback, useState } from "react";
```
Comment (borodayev) [inline src/hooks/useHttp.tsx:1]: Should you delete this file?

**Review by borodayev — CHANGES_REQUESTED** [review]

File: src/elements/modal/signInModal.tsx (hunk 1 above)
Comment (VladMironchuk) [reply src/elements/modal/signInModal.tsx]: fixed

**Pull request merged**